File: PRAK601-2210817220019-DHEA_APRILINDA_UTAMI.py
baris, kolom = map(int, input().split())
matrix =[]
a=0
isi = list(map(int, input().split()))
for i in range (baris):
    matrix.append(isi[a:a+kolom])
    a=a+kolom
for baris in matrix:
    for isi in baris:
        print(isi," ", end="")
    print()

File: PRAK601-2210817220019-DHEA_APRILINDA_UTAMI.c
#include <stdio.h>

    int main()
{
   int baris, kolom, i, j, matrik1[10][10];
 
   scanf("%d %d", &baris, &kolom);

   for (i = 0; i < baris; i++){
      for (j = 0; j < kolom; j++){
  scanf("%d",&matrik1[i][j]);
      }
   }
   
   for (i=0; i<baris; i++){
    for (j=0; j<kolom; j++){
     printf("%3d", matrik1[i][j]);
    }
    printf("\n");
   }
   
}
